lib/mocha.js

```diff
@@ -247,6 +253,28 @@ Mocha.prototype._growl = function(runner, reporter) {
 };
 
 /**
+ * Get require.cache files (node only)
+ *
+ * @api private
+ */
+Mocha.prototype._getRequireCache = function() {
+  return Object.keys(require.cache || {});
```

Comment:
...and because of the above, this will break ES3.  the "jsapi" tests don't get run in the browser.  should they?  not sure.